package.json

```
@@ -27,6 +27,9 @@
     "build:image": "npm run build && docker buildx build --output type=docker --tag pepr:dev .",
     "test": "npm run test:unit && npm run test:journey",
     "test:unit": "npm run gen-data-json && jest src --coverage --detectOpenHandles --coverageDirectory=./coverage --testPathIgnorePatterns='cosign.e2e.test.ts'",
+    "test:int": "npm run test:int:prep && npm run test:int:run",
+    "test:int:prep": "./integration/prep.sh",
+    "test:int:run": "jest --maxWorkers=4 integration",
```

```suggestion
    "test:integration": "npm run test:int:prep && npm run test:int:run",
    "test:integration:prep": "./integration/prep.sh",
    "test:integration:run": "jest --maxWorkers=4 integration",
```

Unless we see ourselves running the command `npm run test:integration:*` a lot locally, we might as well spell it out if it's mostly used in CI